### PR TITLE
Add choose-a-license.md to documentation

### PR DESCRIPTION
**Pull request by s3rrot**
Documentation on research to choose an Open Source license.

### CODE REVIEW

**Comment by s3rrot** [comment]
This PR is ready to be reviewed @yeraydavidrodriguez @cgbosse 

Closes #3

File: doc/choose-a-license.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+
+**Examples** of popular permissive FOSS licenses include the **Berkeley Software Distribution (BSD)**, **MIT** and **Apache** licenses. The BSD and MIT licenses are often mentioned interchangeably due to very similar language and terms that accomplish largely identical goals. The primary intent of these licenses is to allow the use, distribution, and modification of your code for any purpose, while making sure that you as the creator receive credit for your work. The Apache license is similar in effect to the BSD/MIT, but include a license for patents related to the software (this can be desirable or not, depending on the situation).
+
+Below is a list of licenses in descending order **from strong copyleft at the top to completely non-copyleft at the bottom**:
+
+- **GNU General Public License version 3** [GPL-3.0](https://www.gnu.org/licenses/gpl.html)
+- **GNU Affero General Public License version 3** [AGPL-3.0](https://www.gnu.org/licenses/agpl.html)
```
Comment (da2ce7) [inline doc/choose-a-license.md:67]: AGPL is stronger than GPL, should be sorted first.

File: README.md [hunk 2]
```diff
@@ -271,6 +271,8 @@ Not at the moment. It relies on XCode and associated libraries (SpriteKit, Scene
...
 - Can I use it for commercial projects?
 
-Yes, you can, but please, check the License details (TBD) to know the details.
+Yes, you can, but please, check the [License details](LICENSE) to know the details.
 
- 
\ No newline at end of file
+## 9. License
+
+The Boken Engine framework is available under the [Mozilla Public License Version 2.0](https://mozilla.org/MPL/2.0/).
```
Comment (da2ce7) [inline README.md:278]: Include a link to the `doc/choose-a-license.md` document so a reader can also know the rationale and background of this choice.

**Review by da2ce7 — COMMENTED** [review]
Very good work with the Choose a License Document :)

File: doc/choose-a-license.md (hunk 1 above)
Comment (s3rrot) [reply doc/choose-a-license.md:66]: Thanks for the feedback @da2ce7

I've just fixed it.

File: README.md (hunk 2 above)
Comment (s3rrot) [reply README.md:278]: Good idea! I've added an additional text so now they're linked. Thanks!

**Review by yeraydavidrodriguez — APPROVED** [review]
Licence is added, file headers are changed and process documentation is added

**Pull request merged**